Dsalz/Sans-Stitches
# #162573607 Ch implement 11/12/18 feedback

#### What does this PR do?
- Implement feedback gotten from LFA on 11th Dec 2018

#### Description of task to be completed
- Remove password from user object returned
- Trim all data passed to API endpoints
- Validate all parameters in a separate middleware
- Checking the header for 'x-access-token' as well as 'authorization'

#### How should this be manually tested
- Clone the repository
- Checkout into the ```ch-implement-11/12/18-feedback-162573607``` branch
- Install all dependencies
- Run ```npm run dev``` to start server in development mode
- Verify tasks completed above

#### What are the relevant pivotal tracker stories 
- #162573607
